src/Bridge/Laravel/Console/Commands/CreateProviderCommand.php

```diff
@@ -14,7 +14,7 @@ final class CreateProviderCommand extends Command
     public function __construct()
     {
         $this->description = 'Create a new provider';
-        $this->signature = 'app:provider:create {identifier} {name}';
+        $this->signature = 'app:provider:create {--identifier} {--name}';
```

Comment:
Can you please explain what these are:
```{--email= : [optional] The email to use when creating the admin user, required if userId is not specified}```

Obviously only optional if it's actually optional